src/containers/Bounty/SubmissionsAndCommentsCard.module.scss

```diff
@@ -52,8 +52,16 @@
   margin: $xxl-spacing auto;
 }
 
+.borderStyle {
+  li {
+    &:after {
+      border-bottom: $base-border !important;
```

Comment:
why do yo need this important? This shouldn't be required.

Reply:
these are hard coded into the table styles, so i guess i will need to create a prop to handle this

Reply:
yes plz sire.